backend/src/v5/routes/teamspaces/projects/containers/containers.js

```diff
@@ -66,6 +83,69 @@ const appendFavourites = (req, res) => {
 const establishRoutes = () => {
 	const router = Router({ mergeParams: true });
 
+	/**
+	 * @openapi
+	 * /teamspaces/{teamspace}/projects/{project}/containers:
+	 *   post:
+	 *     description: Add a new container to the specified project the user is admin of
+	 *     tags: [Containers]
+	 *     operationId: addContainer
+	 *     parameters:
+	 *       - teamspace:
+	 *         name: teamspace
+	 *         description: Name of teamspace
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+   	 *       - project:
+	 *         name: project
+	 *         description: Project ID
+	 *         in: path
+	 *         required: true
+	 *         schema:
+	 *           type: string
+	 *     requestBody:
+	 *       content:
+   	 *         application/json:
+   	 *           schema:
+	 *             type: object
+	 *             properties:
+	 *               name:
+	 *                 type: string
```

Comment:
missing a lot of fields in the schema.

This should be complete, a mirror to the schema in yup

Bear in mind documentations are for users... they need the full info.

Reply:
also can we add descriptions here.